dec/decode.go

```diff
@@ -83,6 +83,53 @@ func DecompressBuffer(encodedBuffer []byte, decodedBuffer []byte) ([]byte, error
 	}
 }
 
+// DecompressBuffer decompress a Brotli-encoded buffer. Uses decodedBuffer as the destination buffer unless it is too small,
```

Comment:
I think the CI build is failing because [golint complains about this](https://travis-ci.org/kothar/brotli-go/jobs/253199168#L134):

```
< dec/decode.go:86:1: comment on exported function DecompressBufferDict should be of the form "DecompressBufferDict ..."
```